src/WebAppDIRAC/Lib/WebHandler.py

```diff
@@ -17,16 +15,11 @@
 from tornado.web import HTTPError
 from tornado.ioloop import IOLoop
 
-from DIRAC import gLogger, gConfig, S_OK, S_ERROR
+from DIRAC import gLogger, S_OK, S_ERROR
 from DIRAC.Core.Utilities.JEncode import DATETIME_DEFAULT_FORMAT
 from DIRAC.Core.Utilities.Decorators import deprecated
-from DIRAC.Core.DISET.AuthManager import AuthManager
 from DIRAC.Core.DISET.ThreadConfig import ThreadConfig
-from DIRAC.Core.Tornado.Server.TornadoREST import (
-    TornadoREST,
-    authorization,
```

Comment:
Just for note: I added this import for the convenience of writing a handler to get everything you need from one place 
```python
from WebAppDIRAC.Lib.WebHandler import ...
```

Reply:
It was currently only used in one place?

Though in gernerla I agree with the idea of having more intentional imports but, if we do this, we should start setting `__all__` so it's clear what is supposed to be part of the interface.